### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by BlazeWasHere**

### CODE REVIEW

File: test/bridge/slippages-test.ts [hunk 1]
```diff
@@ -0,0 +1,65 @@
+// BEFORE: :file:`src/bridge/slippages.ts`
```
Comment (maozdror) [inline test/bridge/slippages-test.ts:1]: @BlazeWasHere in general, try to strip stuff like this out before a merge

File: test/bridge/slippages-test.ts [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+// % Stmts | % Branch | % Funcs | % Lines | Uncovered Line #s 
+//    72.3 |    54.54 |      50 |    72.3 | 35-37,44-45,52-64
+// AFTER: :file:`src/bridge/slippages.ts`
+// % Stmts | % Branch | % Funcs | % Lines | Uncovered Line #s 
+//     100 |    85.71 |     100 |     100 | 28,32,36                    
+
+
+import { BigNumber } from "ethers";
```
Comment (maozdror) [inline test/bridge/slippages-test.ts:9]: Nitpick: I like to enforce explicit ethers imports for any/all things; `BigNumber` should be imported from `@ethersproject/bignumber`

File: test/bridge/slippages-test.ts (hunk 1 above)
Comment (maozdror) [inline test/bridge/slippages-test.ts:1]: @BlazeWasHere in general, try to strip stuff like this out before a merge

**Review by maozdror — CHANGES_REQUESTED** [review]

File: test/bridge/slippages-test.ts (hunk 1 above)
Comment (maozdror) [reply test/bridge/slippages-test.ts:1]: Rather, while I'd prefer it not be in _code_ files, I do not mind -- actually, I highly encourage it -- if you put coverage details in a different file at the same level as the coverage-checked file.

**Comment by maozdror** [comment]
@BlazeWasHere also, make sure to rebase your branch -- just pushed a tiny change on main dev branch

**Comment by BlazeWasHere** [comment]
> @BlazeWasHere also, make sure to rebase your branch -- just pushed a tiny change on main dev branch

There shouldn't be any merge conflicts now.

**Pull request merged**